src/Edge.hpp

```diff
@@ -1,48 +0,0 @@
-//
-//  Edge.hpp
-//  Delauney Mesh Generation
-//
-//  Created by Ben Rider on 5/9/25.
-//
-
-#include "Point2D.hpp"
-#include <cmath>
-
-
-class Edge{
-
-public:
-    
-    //Change to
-    Point2D a;
-    Point2D b;
-    Point2D midPt;
-    float slope;
-    float magnitude;
-    bool usedInMesh;
```

Comment:
more missing initialization

Reply:
once you add that then you can remove the constructor definition and just let the compiler generate it for you `Edge() = default;`